nf_core/schema.py

```diff
@@ -223,6 +228,68 @@ def validate_default_params(self):
             raise AssertionError("Default parameters are invalid: {}".format(e.message))
         log.info("[green][✓] Default parameters look valid")
```

Comment:
This log comes before we check whether they are valid now.. 😅

Reply:
Ah I see, it's checking that defaults validate in the schema. Ok, needs rewording so that this is clearer..

Reply:
```suggestion
        log.info("[green][✓] Default parameters match schema validation")
```